olm-catalog/serverless-operator/metadata/annotations.yaml

```diff
@@ -0,0 +1,7 @@
+annotations:
+    operators.operatorframework.io.bundle.channel.default.v1: "4.3"
```

Comment:
I am not sure how this works, but isn't it weird that the default channel here is not one of the "channels"? ("operators.operatorframework.io.bundle.channels.v1" only have "4.5" ?